crates/nu-command/src/platform/input/input_.rs

```diff
@@ -50,6 +47,18 @@ impl Command for Input {
                 "default value if no input is provided",
                 Some('d'),
             )
+            .named(
+                "history-file",
+                SyntaxShape::Filepath,
+                "path to a file to read/write history",
```

Comment:
I'm not really sure if this means the current history referenced by $nu.history-path or a separate history file? It should also maybe mention that (I assume) this is a history.txt file vs a history.sqlite3 file.

Information like this level of detail could also be put in an extra_description() function if it's more than just a one-line statement.